src/networkremote/client.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CLIENT_H
```

Comment:
Wrong header guard, it should match the class name in all uppercase.

Reply:
Done

Reply:
Fixed